.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -5,38 +5,109 @@ body:
+      options:
+        - label: I understand that `@opentelemetry/instrumentation-*` packages are not intended to work in bundles apps (with a few exceptions).
+          required: true
+  - type: markdown
+    attributes:
+      value: |
+        ## Bug Reproduction
+        > [!IMPORTANT]
+        >  **If you use a third-party package that re-distributes OpenTelemetry, open the bug ticket with that third party unless you can provide steps to reproduce this with pure OpenTelemetry.**
+        >  Digging into third-party distributions of OpenTelemetry is not in scope for this project.
   - type: textarea
     attributes:
       label: What happened?
-      description: Please provide as much detail as you reasonably can.
+      description: |
+        > [!IMPORTANT]
+        > **Please provide as much detail as you reasonably can.**
+        > We all know the complexities of developing with JavaScript/TypeScript. **Be specific** about your setup and **DO NOT** assume that the
+        > person handling your bug report knows how to use your specific combination of tooling. Always provide build instructions and config files.
       value: |
         ## Steps to Reproduce
-
+          <!--
+            Please consider providing a link to a reproducer repository. It is extremely difficult to exactly reproduce problems
+            and doing so can take multiple hours for us. By providing a reproducer you indirectly contribute back to the project
+            and ensure we can get working on fixing your bug as soon as possible instead of spending precious time asking
+            questions and checking responses. You know your tooling and setup and we know OpenTelemetry JavaScript internals.
+            Let's help each other! :)
+         -->
         ## Expected Result
 
         ## Actual Result
 
         ## Additional Details
+         <!--
+           Anything you can think of that will help us reproduce and understad your problem:
+              - details about the behavior of the bug
+              - code to reproduce, if not provided via a repository above (including setting up any frameworks you may be using, we likely don't use the same tech-stack that you're using on the daily)
+              - which tooling you use and how you're using it
+              - config files for your tooling (typescript, bundlers, ...)
+              - how you run your code (example: `node -r otel.js ./index.js`).
+         -->
+
 
     validations:
       required: true
   - type: textarea
     attributes:
       label: OpenTelemetry Setup Code
-      description: Please provide the code you use to set up OpenTelemetry
+      description: |
+        > [!IMPORTANT]
+        > **If you have provided a reproducer repository above, you may omit this step.**
+
+        **Please provide the code you use to set up OpenTelemetry.**
+        If you use `@opentelemetry/auto-instrumentations-node/register`, please state so here
```

```suggestion
      description: |
        Please provide the code you use to set up OpenTelemetry.
        If you use `@opentelemetry/auto-instrumentations-node/register`, please state so here.
        You may omit this step if you ahve provided a reproducer repository.
```

^^ reformats to have the "OpenTelemetry Setup Code" section formatted like the "package.json" section just below. I don't think the "you may omit this step" note here needs to be called out as "IMPORTANT".

---

applied in https://github.com/open-telemetry/opentelemetry-js/pull/5086/commits/d7245428fff8acda4187c264627cd13569d042a1